fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -109,20 +107,17 @@ public Message getMessage(final FedoraEvent jcrEvent,
         message.setStringProperty(BASE_URL_HEADER_NAME, baseURL);
         message.setStringProperty(USER_HEADER_NAME, jcrEvent.getUserID());
         message.setStringProperty(USER_AGENT_HEADER_NAME, userAgent);
-        message.setStringProperty(PROPERTIES_HEADER_NAME, Joiner.on(',').join(jcrEvent.getProperties()));
+        message.setStringProperty(PROPERTIES_HEADER_NAME, String.join(",", jcrEvent.getProperties()));
 
         LOGGER.trace("getMessage() returning: {}", message);
         return message;
     }
 
     private static String getEventURIs(final Set<Integer> types) {
-        final String uris = Joiner.on(',').join(Iterables.transform(types, new Function<Integer, String>() {
+        final String uris = types.stream()
+                                 .map(x -> REPOSITORY_NAMESPACE + EventType.valueOf(x))
```

Comment:
Right, right. I don't. I always prefer method references, when possible. Think of it, if you like, as a preference for "point-free" style. {grin}